gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/DateTime.java

```diff
@@ -34,7 +32,7 @@
  * @see <a href="https://cloud.google.com/datastore/docs/concepts/entities">Google Cloud Datastore
```

Comment:
I wonder if "Datastore timestamp (represented in micro-seconds)" is still true...